packages/aura-language-server/scripts/build.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
This script should contain the logic in `copy-static-assets.ts` so we can delete that ts file and not have to run it using ts-node by using `copy-static-assets` script in package.json